site/cypress/e2e/cypress-feature/grading_stats.spec.js

```diff
@@ -1,160 +1,68 @@
+            //Change filters back to default
+            ApplyFilter();
         });
 
         it(`${user} view should be accurate for grades.`, () => {
-            cy.get('a[href*="/sample/gradeable/grading_homework/grading/details"]').click();
+            cy.visit(['sample','gradeable','grading_homework','grading','status']);
+            const all_submissions = cy.get('#left-grading-stats').should('exist');
+            all_submissions.should('contain', 'Students who have submitted on time: 59 / 101 (58.4%)');
+            all_submissions.should('contain', 'Current percentage of TA grading done: 30 / 59 (50.8%)');
+            all_submissions.should('contain', 'Section 1: 4 / 9 (44.4%)');
+
+            ApplyFilter();
+            const on_time_submissions = cy.get('#left-grading-stats').should('exist');
+            on_time_submissions.should('contain', 'Students who have submitted: 68 / 101 (67.3%)');
+            on_time_submissions.should('contain', 'Current percentage of TA grading done: 33 / 68 (48.5%)');
+            on_time_submissions.should('contain', 'Section 1: 4 / 9 (44.4%)');
+            //Change filters back to default
+            ApplyFilter();
 
-            /*
-            if (user === 'ta' || user === 'grader') { // Close Grader Responsibilities popup.
-                cy.get('.popup-window .form-title .btn.btn-default.close-button.key_to_click').click();
-            }
-            */
-
-            cy.get('a[href*="/sample/gradeable/grading_homework/grading/status"]').click();
-
-            const text = cy.get('#left-grading-stats');
-            text.should('contain', 'Students who have submitted on time: 68 / 101 (67.3%)');
-            text.should('contain', 'Current percentage of TA grading done: 33 / 68 (48.5%)');
-            text.should('contain', 'Section 1: 4 / 9 (44.4%)');
         });
 
         it(`${user} viewshould be accurate for released grades.`, () => {
-            cy.get('a[href*="/sample/gradeable/grades_released_homework/grading/details"]').click();
-
-            /*
-            if (user === 'ta' || user === 'grader') { // Close Grader Responsibilities popup.
-                cy.get('.popup-window .form-title .btn.btn-default.close-button.key_to_click').click();
-            }
-            */
-
-            cy.get('a[href*="/sample/gradeable/grades_released_homework/grading/status"]').click();
-
-            const text = cy.get('#left-grading-stats');
-            text.should('contain', 'Students who have submitted on time: 71 / 101 (70.3%)');
-            text.should('contain', 'Current percentage of TA grading done: 71 / 71 (100.0%)');
-            text.should('contain', 'Section 1: 10 / 10 (100.0%)');
-            text.should('contain', 'Number of students who have viewed their grade: 49 / 71 (69.0%)');
+            cy.visit(['sample','gradeable','grades_released_homework','grading','status']);
+            const all_submissions = cy.get('#left-grading-stats');
```

Comment:
Instead of using JS variables to save a cypress element, use Cypress variables, like cy.get('#left-grading-stats').as('all_submissions');

and then later do cy.get('@all_submissions').should(...)

Same with other instances of this in this PR.